snuba/datasets/__init__.py

```diff
@@ -93,6 +93,12 @@ def get_prewhere_keys(self) -> Sequence[str]:
         """
         return []
 
+    def get_min_columns(self) -> Sequence[str]:
```

Comment:
Can we have comments on what these two functions are for?

(It's pretty evident in the context of this PR, but might be good for the future)